specification/columns/pricingquantity.md

```diff
@@ -1,6 +1,6 @@
 # Pricing Quantity
 
-The Pricing Quantity represents the volume of a given [*resource*](#glossary:resource) or [*service*](#glossary:service) used or purchased based on the [Pricing Unit](#pricingunit). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not *resource* and *service* consumption.
+The Pricing Quantity represents the volume of a given SKU associated with a [*resource*](#glossary:resource) or [*service*](#glossary:service) used or purchased, based on the [Pricing Unit](#pricingunit). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not *resource* and *service* consumption.
```

Comment:
should we include further disambiguation about why these are separate e.g. " if focuses on pricing and cost reconciling to the associated invoice," ?